test_typing_inspect.py

```diff
@@ -132,19 +179,38 @@ class GetUtilityTestCase(TestCase):
     def test_last_origin(self):
         T = TypeVar('T')
         self.assertEqual(get_last_origin(int), None)
-        self.assertEqual(get_last_origin(ClassVar[int]), None)
+        if WITH_CLASSVAR:
+            self.assertEqual(get_last_origin(ClassVar[int]), None)
         self.assertEqual(get_last_origin(Generic[T]), Generic)
-        self.assertEqual(get_last_origin(Union[T, int][str]), Union[T, int])
-        self.assertEqual(get_last_origin(List[Tuple[T, T]][int]), List[Tuple[T, T]])
+        if EXISTING_UNIONS_SUBSCRIPTABLE:
+            self.assertEqual(get_last_origin(Union[T, int][str]), Union[T, int])
+        base_tp = List[Tuple[T, T]]
+        try:
+            tp = base_tp[int]
+        except TypeError:
+            # python 3.5.3- TypeError: Cannot substitute int for
+            # typing.Tuple[~T, ~T] in typing.List[typing.Tuple[~T, ~T]]
+            pass
```

Comment:
Again I would prefer to use `sys.version_info` here.

Reply:
ok